chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -272,8 +272,23 @@
   realm = nil
 end
 
-template "/opt/dell/crowbar_framework/rainbows.cfg" do
-  source "rainbows.cfg.erb"
+directory "/etc/sysconfig/crowbar" do
+  owner "root"
+  group "root"
+  mode "0700"
+  action :create
+end
+
+
+template "/etc/sysconfig/crowbar/sysconfig.crowbar" do
+  source "sysconfig.crowbar.erb"
+  owner "crowbar"
+  group "crowbar"
+  mode "0644"
+end
+
+template "/opt/dell/crowbar_framework/config/puma.rb" do
+  source "puma.cfg.erb"
```

Comment:
Just NITPICKing but the template file should be the same name as the destination including the erb suffix, e.g. `puma.rb.erb`